src/components/Books.js

```diff
@@ -0,0 +1,13 @@
+import RenderBook from './BookState';
+import AddNewBook from './AddBook';
+
+function Books() {
+  return (
+    <>
+      <RenderBook />
```

Comment:
- [ Optional ] I think it would be better if you render a list of books from this component, currently, the component renders only one book. I would like to suggest that you create an array of `books` and iterate over each `book` in the array using the `map` method as you render each book on the UI. Please see the example code below as a guide on how to achieve this. I highly recommend this change since it will be needed in future milestones of this project.

## Example code
```JavaScript
import RenderBook from './BookState';
import AddNewBook from './AddBook';

function Books() {
 const books = [        
        {
          id: 1,
          cat: 'Action',
          title: 'The Hunger Games',
          author: 'Suzzane Collins',
        },
        {
          id: 2,
          cat: 'Science Fiction',
          title: 'Dune',
          author: 'Frank Herbert',
        },
    ];
  return (
    <>
      { books.map((book) => <RenderBook title={book.title} author={book.author} /> ) }
      <AddNewBook />
    </>
  );
}

export default Books;

```

Reply:
Thank you for reviewing and providing excellent OPTIONAL suggestions. I will take all of them into consideration.